test-cases/repair/decommission-false.yaml

```diff
@@ -0,0 +1,12 @@
+test_duration: 1440
+prepare_write_cmd: "cassandra-stress write cl=QUORUM n=550200300 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=1000 -col 'size=FIXED(200) n=FIXED(5)' -pop seq=1..550200300"
+stress_cmd: "cassandra-stress read cl=QUORUM n=550200300 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=100 -col 'size=FIXED(200) n=FIXED(5)' -pop seq=1..550200300"
```

Comment:
What is the load average during this stress command? 50% load?

Reply:
Will recheck in the new test.